added first 15 shelters to seed.js

ctedtsen opened the pull request.

Jasmine211 commented:
Hi, looks good overall. However, to remain consistent with only storing the state abbreviations in the db, you can either change the 5th shelter's state input to "ME" or you can have a helper function that can do that for you. :)

ctedtsen commented:
Good catch! I fixed that in seed.js so it is now ME instead of Maine. I also added a helper function to check that the input is either a state or a state's abbreviation, then returns the state abbreviation for the given state. This will help to ensure the state is stored in a consistent format
